src/styles/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
You can make it shorter, use [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background)